releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,10 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v6.0.0 - February, 02 2023
+* Upgraded task to use Node 16, previously used Node 10.
+* Fixed a bug that caused an invalid secret Url warning when pulling key vault references in sovereign clouds.
```

Comment:
If capitalizing the U in URL, I'd recommend capitalizing the whole thing since it's an acronym.

Reply:
Also, it may be better to say "loading key vault references".

Reply:
Updated "Url" to "URL" and updated statement to "resolving key vault references"